### ISSUE TITLE
Test if Libgdplus still required

### ISSUE THREAD

**Issue by flytzen** [BrandonPotter/GoogleAuthenticator#109]
When #105 is implemented, test if Libgdplus requirement is obsolete and update docs as required.

Remove some or all of the following code if success:
```
            catch (System.Runtime.InteropServices.ExternalException e)
            {
                if (e.Message.Contains("GDI+") && qrPixelsPerModule > 10)
                {
```

**Comment by flytzen** [comment]
@ahwm Once #114 is merged, do you mind if I push 3.0.0-beta1 to Nuget? It will make it much easier to test Libgdplus.

**Comment by ahwm** [comment]
@flytzen go right ahead.
